providers/install.rb

```diff
@@ -20,12 +20,14 @@
 action :run do
   # Package install
   if node['redisio']['package_install']
-    package "redisio_package_name" do
+    package_resource = package 'redisio_package_name' do
```

Comment:
Thanks, good call! I addressed this with a block instead. Let me know what you think.